_dcp/data/provider/jboss-developer.json

```diff
@@ -466,6 +466,20 @@
         "name":"data_jbossdeveloper_connector",
         "type":"jbossdeveloper_connector"
       }
+    },
+
+    "jbossdeveloper_events": {
+      "description":"Events",
+      "sys_type":"event",
+      "sys_content_content-type" : "text/plain",
+      "persist" : true,
+      "input_preprocessors": [
+      ],
+      "index": {
+        "name":"data_jbossdeveloper_events",
+        "type":"jbossdeveloper_events"
```

Comment:
We need to use singular here as well: `jbossdeveloper_event`
It needs to match with the name of file (without extension): `_dcp/mappings/data_jbossdeveloper_event/jbossdeveloper_event.json`